http/user_agent.ts

```diff
@@ -208,9 +196,6 @@ function mapper(
               target[prop] = match ? match.replace(re, value) : undefined;
             } else {
               const [prop, re, value, fn] = processor;
-              if (!fn) {
-                throw new TypeError("Function must be defined in processor");
-              }
```

Comment:
If `processor` has 4 elements, the 4th element needs to be a function by its typing. So this if-block does nothing.